lib/matplotlib/axes/_axes.py

```diff
@@ -6777,6 +6777,45 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
                           else "list[Polygon]")
             return tops, bins, cbook.silent_list(patch_type, patches)
 
+    def vector(self, dx, dy, x=0., y=0., **kwargs):
```

Comment:
```suggestion
    def vector(self, x, y, dx, dy, **kwargs):
```

- We want to keep the API similar to `arrow()`
- While from a mathemetical point of view, you can have a direction vector only specifying dx, dy; but we need to plot it somewhere and there is no reasonable default. I don't think there is a real use case where you are not interested how the arrow is positioned relative to other plot elements.
- If you have all four parameters x, y, dx, dy is the natural order.